src/components/screens/Login.js

```diff
@@ -35,6 +36,32 @@ export default function Login() {
       const userToken = await api.userLogin(data);
       localStorage.setItem("sessionId", userToken);
       navigate("/main");
+      if ("serviceWorker" in navigator) {
```

Comment:
let's move this code to e.g. `src/service/notifications.js`

so the component will be clean